Makefile

```diff
@@ -18,10 +18,10 @@ export CGO_ENABLED=0
 BINDIR := rootfs/usr/local/bin
 VERSION ?= 0.0.1-$(shell date "+%Y%m%d%H%M%S")
 LDFLAGS := "-s -X main.version=${VERSION}"
-IMAGE_PREFIX ?= deis
+IMAGE_PREFIX ?= deisci
```

Comment:
I think we should still keep this as `deis`. _scripts/deploy.sh handles this: https://github.com/deis/etcd/blob/master/_scripts/deploy.sh#L8